Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY . .
 
 RUN apk update \
   && apk upgrade \
-  && apk add --no-cache git openssh \
+  && apk add --no-cache git openssh curl\
```

Comment:
What changed here? Wondering why we now need curl included and we did not before and the code worked below on line 16.

Reply:
Docker build command failed with error: `curl: not found`. 
Something changed on the latest alpine image, curl is not pre-installed. I didn't find any note on why or how it was changed, but explicitly adding curl resolves the error.

Reply:
Looks good. Newer version of Azure CLI seems to be locking things down. Thx for the update.